tests/test_fixtures/rule_based_profiler/example_notebooks/MultiBatchExample_SqlExample.ipynb

```diff
@@ -61,17 +61,7 @@
    "source": [
     "### Example Database\n",
     "\n",
-    "Imagine we have a database of 12 tables, each corresponding to 1 month of Taxi rider data. \n"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "433ba859-4ae7-4a5c-b4fe-8669d679b620",
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "data_path: str = \"../../../test_sets/taxi_yellow_tripdata_samples/sqlite/yellow_tripdata_2020.db\""
+    "Imagine we have a database of 2 tables, with `yellow_tripdata_sample_2020`, corresponding to all 12 months' `taxi_trip` data for 2020.\n"
```

Comment:
2 tables are referenced, but then it seems like only one is mentioned. Is this an error?